src/pay/components/CheckoutStatus.tsx

```diff
@@ -0,0 +1,15 @@
+import { cn, text } from '../../styles/theme';
+import { useGetCheckoutStatus } from '../hooks/useGetCheckoutStatus';
+import type { CheckoutStatusReact } from '../types';
+
+export function CheckoutStatus({ className }: CheckoutStatusReact) {
+  const { label, labelClassName } = useGetCheckoutStatus();
+
+  return (
+    <div className={cn('flex justify-between', className)}>
+      <div className={text.label2}>
+        <p className={labelClassName}>{label}</p>
+      </div>
```

Comment:
Can we remove one of these Divs?

Reply:
I can try in a follow-up PR, want to keep the scope of this to just renaming the components